src/moshi.py

```diff
@@ -8,6 +8,8 @@
 
 from .common import app
 
+model_cache = modal.Volume.from_name("moshi-model-cache", create_if_missing=True)
```

Comment:
@charlesfrye can I get a sanity check that these volumes are getting used properly?

Reply:
I think you need to point Hugging Face to the Volume still. Simplest would be to set the HF_HOME env var to /cache. We have a bunch of different patterns for this in the examples repo -- pretty much every example that uses huggingface does it some way or another.

Reply:
got it, tysm!